bro-spinn/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/brospinn/SpinnService.kt

```diff
@@ -144,7 +144,7 @@ class SpinnService(
 
     override fun terminate(fail: Fail) {
         MdcUtils.withLogFields(
-            Log.transaksjonId(fail.uuid.let(UUID::fromString))
+            Log.transaksjonId(fail.transaksjonId!!)
```

Comment:
Bør vi få NPE her? Er det ikke bedre å bare gå videre når vi bare ønsker å logge en feil? Bør kanskje heller ta sikte på at transaksjonsId på litt sikt ikke er nullable uansett - den blir jo i hvert fall aldri det lengre dersom den opprettes fra Fail

Reply:
Når jeg tenker meg om så har vi kanskje blitt spart for NPE selv med dagens fallgruve fordi dagens rivers sliter med å lese innkommende feil. Så her har vi et eksempel på at minus og minus blir pluss.

Reply:
Vi har bestemt at vi heller oppretter tilfeldige UUID-er for transaksjonId der den mangler. Det er ikke optimalt, men det ville uansett gått på trynet uten ID-en. Det gjør også at vi kan fjerne `!!` mange steder.